user/enterprise/user-management.md

```diff
@@ -0,0 +1,41 @@
+---
+title: User Management Commands
+layout: en_enterprise
+
+---
+
+These commands are run via the command line on the platform instance.
+
+## User Information
+
+List basic information about users and their status.
+
+`travis users`  - list every user.
+
+`travis users --active` - list every active user.
+
+`travis users --suspended` - list every suspended user.
+
+## Suspend and Unsuspend Users
+
+Suspend or unsuspend users. Builds triggered by suspended users are blocked by `travis-gatekeeper`.
+
+`travis suspend <login>` - suspend a user where `<login>` is the user's GitHub login.
+
+`travis unsuspend <login>` - unsuspend a user where `<login>` is the user's GitHub login.
+
+**Please note**: Using the `suspend` command does not restrict access to the Enterprise platform.
```

Comment:
```suggestion
> **Please note** that using the `suspend` command does not restrict access to the Enterprise platform.
```

Reply:
This one I would keep to be consistent with the other "Please note" below.

Reply:
ok